src/Core/Upload/ResumableUploader.php

```diff
@@ -121,6 +157,11 @@ public function upload()
                     $ex->getCode()
                 );
             }
+            
+            if (is_callable($this->uploadProgressCallback))
```

Comment:
For PSR-2 to be happy, we should put the opening brace on the same line as the control statement.